avaframe/in4Region/splitInputs.py

```diff
@@ -0,0 +1,296 @@
+"""Helper functions for splitting avalanche input data into individual folders"""
+import logging
+import shapefile  # pyshp
+from shapely.geometry import shape, Point, box, MultiPolygon
+from avaframe.in2Trans import ascUtils
+import pathlib
+
+from avaframe.in3Utils.initializeProject import initializeFolderStruct
+
+# create local logger
+log = logging.getLogger(__name__)
+
+def splitInputsMain(inputDir, outputDir, cfg):
+    """Main function for splitting inputs
+
+    Parameters
+    ----------
+    inputDir: pathlib.Path object
+        path to input directory
+    outputDir: pathlib.Path object
+        path to output directory
+    cfg: dict
+        configuration settings
+
+    Returns
+    -------
+    none
+    """
+    # Fetch the necessary input
+    inputRELDir = inputDir / 'REL'
+    inputShp = next(inputRELDir.glob("*.shp"), None)
+    if not inputShp:
+        log.error(f"No shapefile found in {inputRELDir}.")
+        return
+    inputDEM = next(inputDir.glob("*.asc"), None)
```

Comment:
Please mark this with a TODO, we are currently starting to include tif files as well, the glob needs to reflect that